### ISSUE TITLE
wrong evaluation function in lexical analysis model 

### ISSUE THREAD

**Issue by BladeSun** [PaddlePaddle/models#2969]
For each batch, the evalution function saves F1 score based on the accumulated counts from begining. 
The final F1 score is the average of the F1 scores from each batch.

[Link to the code](https://github.com/PaddlePaddle/models/blob/681aca5e3e6de53331795f0d911f8d10e0c78381/PaddleNLP/lexical_analysis/run_ernie_sequence_labeling.py#L183)

**Comment by Halfish** [comment]
You are right, thanks for your feedback, we will fix that later. @BladeSun
